pages/help/[page].tsx

```diff
@@ -0,0 +1,114 @@
+import addSiteLocation from '../../content/help/addSiteLocation.md';
+import { withSites } from '~/lib/sites';
+import ReactMarkdown from 'react-markdown';
+import { useRouter } from 'next/router';
+import Link from 'next/link';
+import { LinkProps } from 'next/link';
+
+export const pages: any = {
+  'add-site-location': addSiteLocation,
+};
```

Comment:
this can probably be done with a [typescript enum](https://www.typescriptlang.org/docs/handbook/enums.html#string-enums)!